FirebaseAuth/Tests/Sample/SwiftApiTests/SwiftApi.swift

```diff
@@ -0,0 +1,458 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import XCTest
+
+import FirebaseAuth
+import FirebaseCore
+
+/// This file tests public methods and enums. Properties are not included.
+/// Each function maps to a public header file.
+
+class AuthApiBuildOnlyTests: XCTestCase {
```

Comment:
Nit: prefer `API` to `Api` (https://www.swift.org/documentation/api-design-guidelines/#conventions under "Follow case conventions"). Same with the file name.